src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -788,8 +788,9 @@ def test_containerapp_managed_service_binding_e2e(self, resource_group):
 
         env_name = self.create_random_name(prefix='containerapp-env', length=24)
         ca_name = self.create_random_name(prefix='containerapp', length=24)
-        mysqlserver = "mysqlflexsb"
-        postgresqlserver = "postgresqlflexsb"
+        mysqlserver = self.create_random_name(prefix='mysqlflexsb', length=15)
```

Comment:
I can't find any public doc for bind a mysql service with --bind. Should we add more example to the _help_?

Reply:
yes i agree. will add!